### PR TITLE
Update training-modules.yaml

### PR DESCRIPTION
**Pull request by 936-BCruz**
Adding the UNIX Shell training

### CODE REVIEW

File: _data/training-modules.yaml [hunk 1]
```diff
@@ -164,3 +164,10 @@
   videos: ''
   webpage: https://hsf-training.github.io/hsf-training-ssh-webpage/
   id: ssh
+- description: A guide through the basics of the file systems and the shell.
+  name: UNIX
+  repository: https://github.com/swcarpentry/shell-novice
+  status: alpha
```
Comment (klieret) [inline _data/training-modules.yaml:170]: this should be `stable`

File: _data/training-modules.yaml [hunk 2]
```diff
@@ -164,3 +164,10 @@
   videos: ''
   webpage: https://hsf-training.github.io/hsf-training-ssh-webpage/
   id: ssh
+- description: A guide through the basics of the file systems and the shell.
+  name: UNIX
```
Comment (klieret) [inline _data/training-modules.yaml:168]: I think `The Unix Shell` would be better here. This is what becomes the title, and unix is a biiiiig topic ;)

**Review by klieret — CHANGES_REQUESTED** [review]

**Comment by klieret** [comment]
Hi @936-BCruz Thanks for starting on this right away ;)  You can address the two change requests by pushing more commits to your branch (not sure if you can do it from the web)

File: _data/training-modules.yaml (hunk 1 above)
Comment (936-BCruz) [reply _data/training-modules.yaml:170]: You mean replacing _status: alpha_, with _status: stable_?

**Review by klieret — APPROVED** [review]

**Pull request merged**